presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -118,6 +118,7 @@ public enum StandardErrorCode
     UNSUPPORTED_ANALYZER_TYPE(0x0001_0029, INTERNAL_ERROR),
     SLICE_TOO_LARGE(0x0001_002A, INTERNAL_ERROR),
     HBO_FAILURE(0x0001_002B, INTERNAL_ERROR),
+    RETRY_QUERY_PLAN_UNCHANGED(0x0001_002C, INTERNAL_ERROR),
```

Comment:
Internal error signifies a bug in Presto, and it's meant to be monitored to provide a high quality SLA.  Doesn't this seem more like an `INSUFFICIENT_RESOURCES` error--Presto is intentionally choosing to fail this query for some reason, analagous to a query using too much memory or running too long?

Reply:
Yeah I can see that being a better fit in this case